### PR TITLE
Metadata schema update

### PR DESCRIPTION
**Pull request by rexwangcc**
### Purpose
_Please explain the purpose of this PR and include links to any GitHub issues that it fixes:_

- https://broadinstitute.atlassian.net/jira/software/projects/GH/boards/530?selectedIssue=GH-129

Previously we weren’t creating the metadata in the correct way, which caused some problems in the downstream services. See:

https://github.com/HumanCellAtlas/metadata-schema/issues/664

Ingest and the HCA metadata team has finished the updates on their sides and plan to promote it on 03/14/2019, we need to respond to that and deploy the change to integration, otherwise the DCP int test will break.

https://github.com/HumanCellAtlas/metadata-schema/pull/875

---
### Changes
_Please list out what major changes were made in this PR to address the issue:_

- Remove the `outputs` from the `analysis_process`.
- Update all of the metadata schema versions for adapter WDLS.
- Update the submit WDL.

---
### Review Instructions
_Please provide instructions about how should a reviewer test/verify the changes in this PR:_

1. Follow the changes in the `pipeline-tools` package first, check the logic.
2. Check if the changes in WDLs comply with the changes in the Python package.
3. https://job-manager.caas-prod.broadinstitute.org/jobs/1b40c71f-c893-4134-9a9d-56cc657fbe21
4. Click into the Submit subworkflow.
5. Enter the execution directory of the task call-create_submission by clicking into the "Folder" icon on the right.
6. Make sure there is no outputs field in analysis_process.json.
7. Make sure that the outputs.json exists.

---
### PR Checklist
_Please ensure the following when opening a PR:_

- [ ] This PR added or updated tests.
- [ ] This PR updated docstrings or documentation.
- [ ] This PR applied code style guidelines:
    - [Google style docstrings](https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html#example-google) Python code.
    - Mint WDL style guide for WDL.
- [ ] This PR considered generalizability beyond our own use case.

---
### Follow-up Discussions
_Please append follow-up discussions and issues during the review process below:_

- No follow-up discussions.

### CODE REVIEW

File: adapter_pipelines/submit.wdl [hunk 1]
```diff
@@ -103,6 +103,7 @@ task create_submission {
       --submit_url ${submit_url} \
       --analysis_process_path analysis_process.json \
       --analysis_protocol_path analysis_protocol.json \
+      --outputs_file_path outputs.json \
```
Comment (samanehsan) [inline adapter_pipelines/submit.wdl]: What do you think about also adding outputs.json as an output of this task? https://github.com/HumanCellAtlas/pipeline-tools/blob/5b587eda87ff949664eae3031d87e05d4e3f898a/adapter_pipelines/submit.wdl#L117

**Comment by samanehsan** [comment]
Please update the unit tests as well! :octocat:

File: adapter_pipelines/Optimus/adapter.wdl [hunk 2]
```diff
@@ -139,7 +139,7 @@ workflow AdapterOptimus {
   Int max_cromwell_retries = 0
   Boolean add_md5s = false
 
-  String pipeline_tools_version = "v0.47.0"
+  String pipeline_tools_version = "rex-metadata-schema-update"
```
Comment (ApChagi) [inline adapter_pipelines/Optimus/adapter.wdl]: this temporary for dev purposes?

Comment (rexwangcc) [reply adapter_pipelines/Optimus/adapter.wdl]: Yeah, due to the nature of this repo (a mixture of dockerized Python package and WDLs that refer to the package in the command blocks), every time we modify the Python package in this repo, we have to follow this special process:
- Change the Python package and use the `branch_name` for all of the `pipeline_tools_version` that were previously semantic (e.g. `v0.47.0`)
- Get approvals on your PR
- Replace the `branch_name` with `semantic version + 1` (e.g.  `v0.48.0`)
- Merge the PR and release the repo/Python Package with `semantic version`. (e.g.  `v0.48.0`)

File: adapter_pipelines/submit.wdl (hunk 1 above)
Comment (rexwangcc) [reply adapter_pipelines/submit.wdl]: Good catch, I meant to do that but ended up forgetting about it! Will do.

**Comment by rexwangcc** [comment]
The Integration test will fail at the validation step, since this metadata schema update has not been promoted to integration Ingest yet. 

In the meantime, the QA instructions for this PR would be:
1. https://job-manager.caas-prod.broadinstitute.org/jobs/1b40c71f-c893-4134-9a9d-56cc657fbe21
2. Click into the **Submit** subworkflow.
3. Enter the execution directory of  the task **call-create_submission** by clicking into the "Folder" icon on the right.
5. Make sure there is no `outputs` field in `analysis_process.json`.
6. Make sure that the `outputs.json` exists.

**Comment by rexwangcc** [comment]
Merging this PR in response to the metadata schema updates happening at the same time.

**Pull request merged**